### PR TITLE
Add Pingdom "How to" Documentation

### PR DESCRIPTION
**Pull request by jasonBirchall**
**Overview**
---
This PR connects to https://github.com/ministryofjustice/cloud-platform/issues/756 and finishes the story around users creating Pingdom checks via the environments pipeline. 

The intention of this document is to provide users with instructions on how to create Pingdom checks via the cloud-platform-environments pipeline.

### CODE REVIEW

File: source/documentation/monitoring-an-app/how-to-create-pingdom-checks.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+ }
+```
+**Note**: You'll need to include the `provider "pingdom"` and `terraform` blocks either in this file or in a `main.tf` file. 
+
+Once reviewed and merged to master, the pipeline will create your check in the MoJ Pingdom account.
+
+This basic check simply checks the `host`/`url` returns a 200 every minute. When six consecutive checks fail it triggers an alarm. As `publicreport` is set to true, you can view the status of your check by visiting the [public status page](http://pingdom.service.dsd.io).
+
+To fully understand all common and uncommon attributes of a check, you can visit the [Pindom community provider](https://github.com/russellcardullo/terraform-provider-pingdom#pingdom-check).
+
```
Comment (digitalronin) [inline source/documentation/monitoring-an-app/how-to-create-pingdom-checks.md]: I would recommend making this sentence simpler. Something like:

> This page explains all the attributes used in the check.

File: source/documentation/monitoring-an-app/how-to-create-pingdom-checks.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+   tags                     = "businessunit_platforms,application_prometheus,component_healthcheck,isproduction_true,environment_prod,                              infrastructuresupport_platforms"
+   probefilters             = "region:EU"
+   publicreport             = "true"
+ }
+```
+**Note**: You'll need to include the `provider "pingdom"` and `terraform` blocks either in this file or in a `main.tf` file. 
+
+Once reviewed and merged to master, the pipeline will create your check in the MoJ Pingdom account.
+
+This basic check simply checks the `host`/`url` returns a 200 every minute. When six consecutive checks fail it triggers an alarm. As `publicreport` is set to true, you can view the status of your check by visiting the [public status page](http://pingdom.service.dsd.io).
```
Comment (digitalronin) [inline source/documentation/monitoring-an-app/how-to-create-pingdom-checks.md]: I would make this a bit more explicit, by adding the full example here. Something like:

> This basic check simply checks that the `host`/`url` (in this case; `https://prometheus.apps.cloud-platform-live-0.k8s.integration.dsd.io/-/healthy`) returns a 200 every minute (`resolution = 1` minute).  When six (`sendnotificationwhendown = 6`) consecutive checks fail it triggers an alarm. As `publicreport` is set to true, you can view the status of your check by visiting the [public status page](http://pingdom.service.dsd.io), where this check would appear with the name "Prometheus - live-0 - cloud-platform - Healthcheck".

File: source/documentation/monitoring-an-app/how-to-create-pingdom-checks.md (hunk 1 above)
Comment (digitalronin) [reply source/documentation/monitoring-an-app/how-to-create-pingdom-checks.md]: One more point - `tags` contains a list of tags, and that's all the [documentation page](https://github.com/russellcardullo/terraform-provider-pingdom#pingdom-check) tells us about that.

Why *this* list of tags? What tags should the developer use, and why? If there are tags that they should leave alone, we should tell them. If they want to add their own tags, we should explain what they can do, and why they might want to.

File: source/documentation/monitoring-an-app/how-to-create-pingdom-checks.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+To fully understand all common and uncommon attributes of a check, you can visit the [Pindom community provider](https://github.com/russellcardullo/terraform-provider-pingdom#pingdom-check).
+
+#### Adding Slack notification
+You can enable the option to send a failing alert to Slack via a webhook by simply adding Pingdom integration id. Unfortunately, you need access to the console to first create the [Pingdom Slack](https://mojdt.slack.com/apps/A0F814AV7-pingdom?next_id=0) webhook and then [Pingdom](https://my.pingdom.com) to create the integration id. 
+
```
Comment (digitalronin) [inline source/documentation/monitoring-an-app/how-to-create-pingdom-checks.md]: > Unfortunately, you need access to the console

Which console? Would this be clearer as; "You need administrator permissions to manage the mojdt slack account..." or, does this mean something else?

Also, that's not "unfortunate" - it's very deliberate ;)

File: source/documentation/monitoring-an-app/how-to-create-pingdom-checks.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+    sendnotificationwhendown = 6
+    notifyagainevery         = 0
+    url                      = "/-/healthy"
+    encryption               = true
+    port                     = 443
+    tags                     = "businessunit_platforms,application_prometheus,component_healthcheck,isproduction_true,environment_prod,                                          infrastructuresupport_platforms"
+    probefilters             = "region:EU"
+    publicreport             = "true"
+    integrationid            = [1000]
+  }
```
Comment (digitalronin) [inline source/documentation/monitoring-an-app/how-to-create-pingdom-checks.md]: I think this should be `integrationids`

Also, I'd be more explicit, like this;

> your check will now appear as follows (assuming you were given 1000 as the integration id)

File: source/documentation/monitoring-an-app/how-to-create-pingdom-checks.md (hunk 1 above)
Comment (jasonBirchall) [reply source/documentation/monitoring-an-app/how-to-create-pingdom-checks.md]: Totally agree with the points made here. Document has been amended under ec67bfe.

File: source/documentation/monitoring-an-app/how-to-create-pingdom-checks.md (hunk 2 above)
Comment (jasonBirchall) [reply source/documentation/monitoring-an-app/how-to-create-pingdom-checks.md]: Really good example, I've replaced the paragraph with the one mentioned here in ec67bfe.

File: source/documentation/monitoring-an-app/how-to-create-pingdom-checks.md (hunk 3 above)
Comment (jasonBirchall) [reply source/documentation/monitoring-an-app/how-to-create-pingdom-checks.md]: Yep, it wasn't overly clear. This has been addressed in ec67bfe.

File: source/documentation/monitoring-an-app/how-to-create-pingdom-checks.md (hunk 4 above)
Comment (jasonBirchall) [reply source/documentation/monitoring-an-app/how-to-create-pingdom-checks.md]: :+1: Spot on.

**Comment by digitalronin** [comment]
Looks great!

**Pull request merged**